metisp/pymetis/src/pymetis/dataitems/hci/hci.py

```diff
@@ -459,3 +462,67 @@ class NCvcSciSnr(BandLmMixin, CgrphAppMixin,  SciSnr):
 
 class NCvcPsfMedian(BandLmMixin, CgrphAppMixin, PsfMedian):
     pass
+
+
+###
+
+
+class LmAppSciCentred(BandLmMixin, CgrphAppMixin, SciCentred):
+    _name_template = r"{band}_APP_SCI_CENTRED"
```

Comment:
If this is indeed something that can be parameterized by `cgrph`, the name template should most likely read `r'{band}_{cgrph}_SCI_CENTRED`